src/privileged/notificationBar/schema.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "namespace": "notificationBar",
+    "description":
+      "A part of the study add-on that needs to be chrome-privileged",
+    "functions": [
+      {
+        "name": "show",
+        "type": "function",
+        "description": "Show the notification bar to the user.",
+        "async": true,
```

Comment:
nit: The implementation is not an async function.

Reply:
This is necessary for the API to work, I tried removing it.